[New Product] Apache Subversion (#5567)
captn3m0/endoflife.date: products/apache-subversion.md

    lts: true
    releaseDate: 2013-06-18
    eol: true
    latest: "1.8.19"
    latestReleaseDate: 2017-08-10

-   releaseCycle: "1.7"
    lts: true
    releaseDate: 2011-10-11
    eol: true
    latest: "1.7.22"
    latestReleaseDate: 2015-08-14

-   releaseCycle: "1.6"
    lts: true
    releaseDate: 2009-03-20
    eol: true
    latest: "1.6.23"
    latestReleaseDate: 2013-05-31

-   releaseCycle: "1.5"
    lts: true
    releaseDate: 2008-06-19
    eol: true
    latest: "1.5.9"
    latestReleaseDate: 2010-12-22

-   releaseCycle: "1.4"
    lts: true
    releaseDate: 2006-09-10
    eol: true
    latest: "1.4.6"
    latestReleaseDate: 2007-12-20

-   releaseCycle: "1.3"
    lts: true
    releaseDate: 2005-12-30
    eol: true
    latest: "1.3.2"
    latestReleaseDate: 2006-05-31

-   releaseCycle: "1.2"
    lts: true
    releaseDate: 2005-05-21
    eol: true
    latest: "1.2.3"
    latestReleaseDate: 2005-08-25

-   releaseCycle: "1.1"
    lts: true
    releaseDate: 2004-09-29
    eol: true
    latest: "1.1.4"
    latestReleaseDate: 2005-04-01

-   releaseCycle: "1.0"
    lts: true
    releaseDate: 2004-02-23
    eol: true
    latest: "1.0.9"
    latestReleaseDate: 2004-10-13

---

> > [Apache Subversion](https://subversion.apache.org/) is a full-featured version control system
> > originally designed to be a better CVS. Subversion has since expanded beyond its original goal
> > of replacing CVS, but its basic model, design, and interface remain heavily influenced by that
> > goal. Even today, Subversion should still feel very familiar to CVS users.

Reviewer: CVS hasn't had a release in almost 20 years now, mentioning it feels like a distraction. 

```suggestion
> [Apache Subversion](https://subversion.apache.org/) is an open-source full-featured
> centralized version control system.
```
